frontend/packages/console-app/src/components/quick-starts/loader/QuickStartsLoader.tsx

```diff
@@ -12,17 +14,21 @@ const QuickStartsLoader: React.FC<QuickStartsLoaderProps> = ({ children }) => {
   const [allowedQuickStarts, setAllowedQuickStarts] = React.useState<QuickStart[]>([]);
   const [loaded, setLoaded] = React.useState<boolean>(!(quickStarts.length > 0));
   const permissionChecks = React.useRef<{ [name: string]: boolean }>({});
+  const [qs, qsLoaded, qsError] = useK8sWatchResource<QuickStart[]>({
+    kind: referenceForModel(QuickStartModel),
+    isList: true,
+  });
 
   React.useEffect(() => {
-    setQuickStarts(getQuickStarts());
-  }, []);
+    if (qs && qsLoaded && !qsError) setQuickStarts(qs);
```

Comment:
You don't need extra state for this. Just use whatever is returned from the `useK8sWatchResource` hook. No need for this effect.

Reply:
removed it